admin/config/locales/en.yml

```diff
@@ -746,6 +746,21 @@ en:
           save_asset: Save Asset
           tags_label: Tags
           tags_note: 'Comma separated: just, like, this'
+          tags_tooltip_html: |
+            <p>You can specify tags on an asset to mark them for special use cases.</p>
+            <p>
+              <strong>Favicons:</strong> If you want to use an image asset as a favicon for the site,
+              you can add the <code>favicon</code> tag. Additionally, you can specify use for a
+              specific size favicon by tagging it with <code>favicon-HxW</code>. For example, a tag of
+              <code>favicon-32x32</code> will indicate the asset should be used for the favicon when
+              it's size is 32x32.
+            </p>
+            <p>
+              <strong>Open Graph:</strong> You can specify what image asset to use by default for the
+              open graph image of your content pages by tagging it with the <code>og-default</code> tag.
+              When shared on Facebook, that asset will display if there is no open graph image specified
```

Comment:
```suggestion
              When shared on social media, that asset will display if there is no open graph image specified
```

Reply:
is this true? I thought Facebook was the only use for open graph tags.

Reply:
I know Slack also uses it, maybe others.